OpenTrons/opentrons-api
# fix(components, protocol-designer): add hover state for Checkbox component and StyledLabel

# Overview

Fix enabled and hover state background color for both checked and unchecked states of Checkbox component, and add hover state for StyledLabel one-off in SelectPipettes component.

Closes RQA-3655

## Test Plan and Hands on Testing

- create new OT-2 protocol and land on select pipettes step
- select type and volume
- click around and hover on tiprack options and verify that hover state is correct
- hover "Add custom pipette tips" and "Show default tips" links and verify text changes to blue50

https://github.com/user-attachments/assets/4bd1890c-868c-47f9-a255-e8abf6e6e21f


## Changelog

- add hover states

## Review requests

- see test plan

## Risk assessment

low
